script.module.clouddrive.common/addon.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.clouddrive.common" name="Cloud Drive Common Module" version="1.3.9+matrix.0" provider-name="Carlos Guzman (cguZZman)">
+	<requires>
+		<import addon="xbmc.python" version="3.0.0" />
+		<import addon="script.module.dateutil"/>
+		<import addon="script.module.pyqrcode"/>
+	</requires>
+	<extension point="xbmc.python.module" library="/" />
+	<extension point="xbmc.service" library="service.py" start="login" />
```

Comment:
`start="login"` isn't supported in matrix, and is the cause of the Travis-CI failure
https://travis-ci.org/github/xbmc/repo-scripts/builds/747725813#L271